src/components/EResourceSections/PackageContents.js

```diff
@@ -3,35 +3,66 @@ import PropTypes from 'prop-types';
 import { get } from 'lodash';
 import { FormattedMessage } from 'react-intl';
 
-import { Accordion, Badge, MultiColumnList } from '@folio/stripes/components';
+import { Accordion, Badge, Button, ButtonGroup, Layout, MultiColumnList } from '@folio/stripes/components';
 import { Spinner } from '@folio/stripes-erm-components';
 
 import CoverageStatements from '../CoverageStatements';
 import EResourceLink from '../EResourceLink';
+import FormattedUTCDate from '../FormattedUTCDate';
 
 export default class PackageContents extends React.Component {
   static propTypes = {
     data: PropTypes.shape({
+      filterPath: PropTypes.string,
       packageContents: PropTypes.array,
     }),
+    onClickFilterButton: PropTypes.func.isRequired,
   };
 
-  renderOptions = () => (
-    <MultiColumnList
-      contentData={this.props.data.packageContents}
-      interactive={false}
-      columnMapping={{
-        name: <FormattedMessage id="ui-agreements.eresources.name" />,
-        platform: <FormattedMessage id="ui-agreements.eresources.platform" />,
-        coverage: <FormattedMessage id="ui-agreements.eresources.coverage" />,
-      }}
-      formatter={{
-        name: pci => <EResourceLink eresource={pci._object.pti.titleInstance} />,
-        platform: pci => get(pci._object, 'pti.platform.name', ''),
-        coverage: pci => <CoverageStatements statements={pci._object.coverage} />,
-      }}
-      visibleColumns={['name', 'platform', 'coverage']}
-    />
+  columnMapping = {
+    name: <FormattedMessage id="ui-agreements.eresources.name" />,
+    platform: <FormattedMessage id="ui-agreements.eresources.platform" />,
+    coverage: <FormattedMessage id="ui-agreements.eresources.coverage" />,
+    accessStart: <FormattedMessage id="ui-agreements.eresources.accessStart" />,
+    accessEnd: <FormattedMessage id="ui-agreements.eresources.accessEnd" />,
+  }
+
+  columnWidths = {
+    name: 250,
+    platform: 150,
```

Comment:
Should we define a width for `platform`? My default these days is to leave widths undefined because MCL is much better at calculating widths automatically, but if you've seen it consistently sizing it poorly, this makes sense.